includes/snmp.inc.php

```diff
@@ -221,14 +221,30 @@ function snmp_get_multi($device, $oids, $options = '-OQUs', $mib = null, $mibdir
 function snmp_get_multi_oid($device, $oids, $options = '-OUQn', $mib = null, $mibdir = null)
 {
     $time_start = microtime(true);
-
-    if (is_array($oids)) {
+        $oid_limit = get_device_oid_limit($device);
+    if (!is_array($oids)) {
+        $oids = explode(" ", $oids);
+    }
+    $oid_number = count($oids);
+
+    if ($oid_limit < $oid_number) {
+        $data = [];
+        while (!empty($oids)) {
```

Comment:
If you use array_chunk(), you can avoid the if/else and make this a lot simpler.

Reply:
Changed.
There's a problem with "\n" as divider.
Some result may have it inside response, like : 
```
"""
.1.3.6.1.2.1.1.3.0 = 3402322978\n
.1.3.6.1.2.1.1.6.0 = \n
.1.3.6.1.2.1.1.4.0 = \n
.1.3.6.1.2.1.1.5.0 = xxxxxxxxxxxx\n
.1.3.6.1.2.1.1.2.0 = .1.3.6.1.4.1.9.1.1639\n
.1.3.6.1.2.1.1.1.0 = Cisco IOS XR Software (Cisco ASR9K Series),  Version 6.1.3[Default]\r\n
Copyright (c) 2017 by Cisco Systems, Inc.
"""
```